### DISCUSSION TITLE
Barotrauma should have ability to do rotation in Polar coordinate system

### DISCUSSION THREAD

**Discussion by PanmanS** [FakeFishGames/Barotrauma#13562] [Ideas and suggestions]
Basically I think there should be a way to rotate parts as whole rather as individual to make stuff like this more easily done
![mspaint_IpJjZumhdn](https://github.com/FakeFishGames/Barotrauma/assets/42521948/25b5101d-62cf-4ccb-bc64-f13b561a30c0)

**Comment by EebleSheembl** [comment]
Lovely interpretation
